tests/unit/new_detector_test.py

```diff
@@ -0,0 +1,90 @@
+# ----------------------------------------------------------------------
+# Numenta Platform for Intelligent Computing (NuPIC)
+# Copyright (C) 2014-2015, Numenta, Inc.  Unless you have an agreement
+# with Numenta, Inc., for a separate license for this software code, the
+# following terms and conditions apply:
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 3 as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+# See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see http://www.gnu.org/licenses.
+#
+# http://numenta.org/licenses/
+# ----------------------------------------------------------------------
+
+"""Tests scripts.create_new_detector for creating appropriate dirs and files."""
+
+import shutil
+import os
+import unittest
+
+try:
+  import simplejson as json
+except ImportError:
+  import json
+
+from scripts.create_new_detector import createResultsDir, createThresholds
```

Comment:
Please order imports as in the [styling guidelines](https://github.com/numenta/nupic/wiki/Python-Style-Guide#coding-style).